samples/performance/subpasses/subpasses_tutorial.md

```diff
@@ -117,7 +116,7 @@ In practice, their [image usage](https://www.khronos.org/registry/vulkan/specs/1
 
 **Do**
 
-* Use sub-passes.
+* Use subpasses.
 * Use a 128-bit G-buffer budget for color.
```

Comment:
This limit appears to be Mali-specific. Maybe reword to a more general "aim to keep your tile buffer budget small"